src/openrct2-ui/input/KeyboardShortcut.cpp

```diff
@@ -814,6 +823,201 @@ static void shortcut_scale_down()
     context_update_cursor_scale();
 }
 
+// Tile inspector shortcuts
+static void shortcut_insert_corrupt_element()
+{
+    rct_window* w = window_find_by_class(WC_TILE_INSPECTOR);
+    if (w != nullptr && !widget_is_disabled(w, WC_TILE_INSPECTOR__WIDX_BUTTON_CORRUPT)
```

Comment:
@Broxzier The current PR exposes all these widget ids through `Window.h`. Is there a better way of accessing the status of these buttons?

Reply:
So I based it on how some of the other window-specific widgets are accessed, such as staff tools or the scenery eyedropper. If you look above my changes in Window.h, you'll see other widgets accessed in a similar way.